### DISCUSSION TITLE
How to use SVG's in next.js from mui?

### DISCUSSION THREAD

**Discussion by jk0ehn** [neuefische/questions#224] [Web]
Is there a recommended way?

**Comment by julialoeschel** [comment]
Hello there, 
we found the following tips for mui: https://medium.com/nextjs/how-to-use-material-ui-with-nextjs-and-react-18-6c054ceacf77
and , if you decide for another <svg> , check this: https://www.codeconcisely.com/posts/using-svg-in-nextjs/

**Reply by jk0ehn** [reply]
Is there an other way, by not overwrite -app.js ?
